doc/sphinx/source/sitemap.rst

```diff
@@ -3,6 +3,6 @@
 .. _sitemap.rst:
  
 .. toctree::
-   :includehidden:
- 
+   :includehidden: 
```

Comment:
Might want to just revert this change since it does not look like anything really changed.

Reply:
Yeah, weird merge diff from fixing this yesterday for the OTC folks.